fritzconnection/tests/test_fritzhosts.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from ..lib.fritzhosts import (
+    FritzHosts,
+)
+
+
+@pytest.fixture()
+def mesh_topology_fixture():
+    return {'schema_version': '1.9',
+            'nodes': [{'uid': 'n-1',
+                'device_name': 'fritz.box',
+                'device_model': 'FRITZ!Box Model',
+                'device_manufacturer': 'AVM',
+                'device_firmware_version': '111.11.11',
+                'device_mac_address': '11:11:11:11:11:11',
+                'is_meshed': True,
+                'mesh_role': 'master',
+                'meshd_version': '2.25',
+                'node_interfaces': []
+               }]}
+
+
+def test_get_mesh_topology_info(mesh_topology_fixture):
+    """
+    Simple test of the topology_info function.
+    """
+    host = FritzHosts()
+    result = host.get_mesh_topology_info(topology=mesh_topology_fixture)
```

Comment:
I think that doesn't work:
```
        if topology is None:
            try:
>               topology = self.get_mesh_topology()
E               AttributeError: 'NoneType' object has no attribute 'get_mesh_topology'
```

Reply:
If we use `host = FritzHosts()` we don't necessarily connect to a fritzbox, do we?

Reply:
The first test with a given topology will work, but the second one with `topology=None` will fail. I didn't saw this at first. For that some mocking seems to be needed. Because instanciating `FritzHosts()` will connect to the box.